warehouse/query-core/src/test/java/datawave/query/IndexHoleQueryTest.java

```diff
@@ -15,7 +15,7 @@
 import org.junit.ClassRule;
 import org.junit.Test;
 
-import datawave.query.config.IndexHole;
+import datawave.query.config.IndexValueHole;
 import datawave.query.exceptions.FullTableScansDisallowedException;
```

Comment:
Rename test class to IndexValueHoleQueryTest?

Reply:
done